lib/Transformations.cpp

```diff
@@ -19,6 +19,24 @@ namespace CIL {
         }
     }
 
+    void changeContrast(ImageInfo& img, const double contrast)
+    {
+        double temp = 259 * (contrast + 255) / (255 * (259 - contrast));
+        for (auto px : img)
+        {
+            for (auto i = 0; i < px.numComponents(); i++)
```

Comment:
Should we add any extra conditions here for the alpha channel?

Reply:
I am not changing the alpha channel anymore